packages/dapp/src/store/reducer.ts

```diff
@@ -128,6 +130,15 @@ export const reducer = (state: State, action: Action): State => {
         signIn: action.payload.signIn,
         signOut: action.payload.signOut
       };
+    case 'REGISTER_IPFS_NODE':
+      return {
+        ...state,
+        node: action.payload
+      };
+    case 'UNREGISTER_IPFS_NODE':
+      return {
+        ...state
```

Comment:
here is the error. You are do not updating the state at all.
You must reset the property explicitly.
```
return {
  ...state,
  node: undefined
};
```
or
```
delete state.node;
return state;
```